packages/fxa-graphql-api/package.json

```diff
@@ -64,7 +64,7 @@
     "@types/chance": "^1.1.2",
     "@types/convict": "^5.2.2",
     "@types/graphql": "^14.5.0",
-    "@types/jest": "^29.2.5",
+    "@types/jest": "29.2.5",
```

Comment:
Can we push these down to the root package.json so it's consistent across all packages/libs?

Reply:
I think payments-server is stuck on Jest 27 until they eject create-react-app (or more likely) replace with the in-progress SubPlat 3 work.

Reply:
Thanks for the feedback!  Yes, well do on the types.  I had already moved typescript down to root 👍

Reply:
Sounds good to me!